deepchecks/vision/batch_wrapper.py

```diff
@@ -75,13 +89,19 @@ def __getitem__(self, index: int):
 
     def __len__(self):
         """Return length of batch."""
-        return len(self._batch)
+        dataset = self._context.get_data_by_kind(self._dataset_kind)
```

Comment:
I'm not sure this is the best way to get batch length, maybe we should use `len(batch_to_images(batch))` 
@noamzbr WDYT?

Reply:
batch_to_images not always implemented.
fixed case of sampled data